.gitignore

```diff
@@ -152,3 +152,6 @@ configs/local/default.yaml
 
 # Aim logging
 .aim
+
+# Network File System (NFS) files
+.nfs*
```

Comment:
Is this needed for this PR or was it added accidently?

Reply:
Whoops, added accidentally. I'll remove it.

Reply:
Should be removed now. Just rebased and pushed.